packages/data/src/world/oot/overworld.yml

```diff
@@ -799,6 +815,10 @@
     "Goron City Shortcut": "true"
     "Zora River": "can_dive_small || hookshot_anywhere"
     "Lost Woods Rupee Arrow": "is_child && (can_dive_small || hookshot_anywhere || can_boomerang)"
+    "Lost Woods Lost North": "true"
+    "Lost Woods Lost East": "true"
+    "Lost Woods Lost South": "true"
+    "Lost Woods Lost West": "true"
```

Comment:
You cannot access East in the first part of Lost Woods (before Mido). East can only be accessed in Lost Woods Deep. All four should be added to Lost Woods Deep while doing since adult can't access the ones in the front if coming from SFM